murdock/main.py

```diff
@@ -82,6 +81,12 @@
 )
 
 
+@app.on_event("startup")
+async def startup():
+    murdock.instrumentator.instrument(app).expose(app)
```

Comment:
Why not simply move this to the murdock.init method ?

Reply:
And pass the `app` as argument to the `murdock.init` method?

Reply:
And pass the app as argument to the murdock.init method?

Reply:
ah you are right, this has to be here...